keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListenerTests.kt

```diff
@@ -206,7 +204,8 @@ internal class ArtifactListenerTests : JUnit5Minutests {
     .build()
 
   data class SyncArtifactsFixture(
-    val artifact: DeliveryArtifact,
```

Comment:
I added tests for syncing docker image versions as well here, because that's a big change in this PR.